README.md

```diff
@@ -30,19 +31,33 @@ query and filter sample data via REST.
 
 #### Application information
 
-- Name: `loopback-example-model-relations`
-- Directory to contain the project: `loopback-example-model-relations`
-
 ```
 $ slc loopback loopback-example-model-relations
 ... # follow the prompts
+- Name: `loopback-example-model-relations`
+- Directory to contain the project: `loopback-example-model-relations`
 $ cd loopback-example-model-relations
 ```
+### Create the datasource
+
+```
+$ slc loopback:datasource
+... # follow the prompts, choose `other` to define custom connctor
+```
+- Name: `transient`
+- Connector: `other`
+  - Name: `transient`
```

Comment:
thank you Simon, I changed the order of command and input data because I thought the tutorial would better to teach the command first then provide the data. But now I think you are right, a user should know what they gonna do before they really do it. I will change all these things back.